Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM node:10.16-alpine
```

Comment:
do we need nodejs there ?

Reply:
No, I will update to use an alpine-only image.